homework/index.js

```diff
@@ -30,13 +30,188 @@
+            });
+            createAndAppend('div', contDiv, {
+              id: `${item.login}Badge`,
+              class: 'cont-badge',
+              text: item.contributions,
+            });
+          });
+        }
+      },
+    );
+  }
+
+  function updateContSection(selected) {
+    const contsDiv = document.getElementById('contsDiv');
+    contsDiv.innerHTML = '';
+    createAndAppend('p', contsDiv, { id: 'cont-header', text: 'Contributors' });
+    const contsLists = createAndAppend('ul', contsDiv, { id: 'conts-list' });
+
+    fetchJSON(
+      `https://api.github.com/repos/HackYourFuture/${selected}/contributors`,
+      (err, data) => {
+        if (err) {
+          createAndAppend('div', root, { class: 'alert-error' });
+        } else {
+          data.forEach(item => {
+            const contListItem = createAndAppend('li', contsLists, {
+              id: `${item.login}ListItem`,
+              class: 'cont-li',
+            });
+
+            createAndAppend('img', contListItem, {
+              id: `${item.login}Img`,
+              class: 'cont-avatar',
+              src: item.avatar_url,
+            });
+
+            const contDiv = createAndAppend('div', contListItem, {
+              id: `${item.login}Div`,
+            });
+
+            createAndAppend('a', contDiv, {
+              id: `${item.login}Name`,
+              class: 'cont-name',
+              text: item.login,
+              href: item.html_url,
+              target: '_blank',
+            });
+            createAndAppend('div', contDiv, {
+              id: `${item.login}Badge`,
+              class: 'cont-badge',
+              text: item.contributions,
+            });
+          });
+        }
+      },
+    );
+  }
+
+  function updateInfoSection(selected, data) {
+    const selectedData = getSelectedData(selected, data);
+
+    const repoName = document.getElementById('repoName');
+    repoName.innerText = '';
+    repoName.innerText = `Repository: ${selectedData.name}`;
+    repoName.href = selectedData.html_url;
+    repoName.target = '_blank';
+
+    const desc = document.getElementById('desc');
+    desc.innerText = '';
+    desc.innerText = `Description: ${selectedData.description}`;
+
+    const forks = document.getElementById('forks');
+    forks.innerText = '';
+    forks.innerText = `Forks: ${selectedData.forks}`;
+
+    const updatedAt = document.getElementById('updated_at');
+    updatedAt.innerText = '';
+    updatedAt.innerText = `Updated: ${selectedData.updated_at}`;
+  }
+
   function main(url) {
     fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
       if (err) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        const repoName = data
+          .map(repo => repo.name)
+          .sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase()));
```

Comment:
When you use `localeCompare`, you don't also have to use `toLowerCase`. `localeCompare` has its own means to compare casing, based on the language.